pkg/scheduler/framework/plugins/volumerestrictions/volume_restrictions.go

```diff
@@ -158,10 +158,27 @@ func haveOverlap(a1, a2 []string) bool {
 	return false
 }
 
+// return true if there is no conflict checking targets.
+func noneConflictVolume(v v1.Volume) bool {
```

Comment:
Make it return true if we need to check.

```suggestion
func needsRestrictionsCheck(v *v1.Volume) bool {
```

Reply:
We discussed about using pointer or struct previously in https://github.com/kubernetes/kubernetes/pull/114898#discussion_r1137295540